src/azul/plugins/metadata/hca/transform.py

```diff
@@ -327,7 +328,8 @@ def _donor(self, donor: api.DonorOrganism) -> MutableJSON:
             'biological_sex': donor.sex,
             'genus_species': sorted(donor.genus_species),
             'diseases': sorted(donor.diseases),
-            'organism_age': donor.organism_age,
+            'organism_age': f'{donor.organism_age} {donor.organism_age_unit}',
```

Comment:
What if one or both of the age values are empty/None?
```suggestion
            'organism_age': ' '.join(x for x in (donor.organism_age, donor.organism_age_unit) if x),
```

Reply:
I had to revert this change since it cause failures for the `TestResponse` suite. Is there a reason why you think the f string is undesirable?

Reply:
What is the expected `organism_age` value if both `donor.organism_age` and `donor.organism_age_unit` are `None`? With the f-string implementation you would get the string "None None"